content/2017-01-13_jako-holka-v-brne.md

```diff
@@ -0,0 +1,56 @@
+Title: Jako holka v Brně jsi už napůl programátorka
+Date: 2017-01-13
+Author: Lucie Karmová
+About: Lucka pracuje jako úřednice v Brně a spravuje dotační programy pro mimořádně nadané studenty z Jihomoravského kraje. Fascinuje jí programátorská komunita v Brně, a proto se přihlásila na kurz PyLadies v Brně.
+
+**Žiju s programátorem, půlka mých přátel jsou programátoři, na počítači mi běží Fedora, mám za sebou několik neslavných programovacích začátků a nakonec jsem narazila na poklad. Mohla jsem si jen vymýšlet důvody, proč se NEučit programovat. Nakonec ale nebylo vyhnutí a přihlásila jsem se do programování v Pythonu pro holky – PyLadies.**
+**Co všechno jsem kdy nedokončila? Learn Python the Hard Way od Zeda A. Shawa, CodeAcademy nebo Programming for Everybody na Courseře. Taky jsem uvízla v polovině nulté (ano, nulté) série Korespondenčního semináře z informatiky FI MU. Celý jsem odchodila a pobrala snad jenom kurz SQLite na KISKu. Pak okolo mne prolétla registrace na PyLadies a bylo jasné, že to je to pravé. Přihlášku jsem poslala, naslibovala hory doly ohledně své ochoty makat, vzali mě a už to jelo.**
+
+
+![linux]({filename}/images/01.png)
+
+###Když v Brně hodíte kamenem, trefíte programátora
+Pro holky v Brně existuje jedinečný časoprostor. Tohle město je významné IT centrum a vývojáři se vyvažují zlatem. Zároveň jde o totálně mužské prostředí a mám pocit, že z toho samotní programátoři nejsou moc šťastní. Všichni se můžou přetrhnout, aby holkám umetli cestičku k programování. Holky v IT jsou prostě téma. 
```

Comment:
Zároveň je to téměř výhradně mužské prostředí.
(nevím, teda, jestli fakt není ani jedna žena :D takže proto "téměř". Klidně ale vymaž.)